brave/brave-core
# Update adblock-rust to v0.8.7

Resolves https://github.com/brave/brave-browser/issues/36223



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Enable the checkbox for `Adguard Russian` in `brave://settings/shields/filters`
- Add `pravda.com.ua,epravda.com.ua##+js(remove-node-text, script, \,mr=function(r\,)` to the custom filters in `brave://settings/shields/filters` and save
- Navigate to `pravda.com.ua` and `epravda.com.ua`
- The sites should be functional and not show any ads, anti-adblock, unhandled whitespace, etc.
